requirements.txt

```
@@ -1,5 +1,5 @@
 # generated from manifests external_dependencies
 dataclasses
-odoorpc
-openupgradelib
+git+https://github.com/OCA/odoorpc.git@master
+git+https://github.com/OCA/openupgradelib.git@master
```

Assuming we want to use the master branch in tests, these should go in `test-requirements.txt` with this form:

```suggestion
odoorpc @ git+https://github.com/OCA/odoorpc.git@master
openupgradelib @ git+https://github.com/OCA/openupgradelib.git@master
```

`test-requirements.txt` are installed by MQT in PIP and OCA modes and will be installed by GitHub actions, and are very useful to cleanly separate test dependencies from runtime dependencies.